com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/pipeline/TestTextureProfileCompressor.java

```diff
@@ -0,0 +1,22 @@
+package com.defold.extension.pipeline.texture;
+
+public class TestTextureProfileCompressor implements ITextureCompressor {
```

Comment:
These needs to be in their own files due to the package. Java throws instantiate exceptions otherwise